### DISCUSSION TITLE
Rename the table groups again

### DISCUSSION THREAD

**Discussion by bschmalhofer** [RotherOSS/otobo#1215] [Ideas]
The table **groups** has been renamed by several projects because `groups` is a keyword in MySQL 8.

- OTRS 8: groups_table
- OTOBO 10.0: groups_table
- Znuny 6.1: permission_groups

That divergence is not nice. Maybe it makes sense that OTOBO sides with Znuny instead of OTRS.

**Comment by stefanhaerter** [comment]
Is this worth being converted to an issue?

**Comment by bschmalhofer** [comment]
I'd leave this as a discussion, maybe even close the discussion at that ship has sailed. 

Nonetheless, I would have preferred it Znuny and OTOBO would have kept that table name in sync.

**Comment by eyazi** [comment]
Besides that, there are the tables: group_customer, group_customer_user, group_role, group_user, role_user, and roles.
I think that these tables should be included in the decision to make sure they are aligned next to each other – whatever the decision is.
